### PR TITLE
bngp-3171-doj-seed-application

### PR DESCRIPTION
**Pull request by shaundefra**
seeds test data for the developer journey

### CODE REVIEW

File: packages/webapp/src/routes/__tests__/test/seed-developer-data.spec.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { submitGetRequest } from '../helpers/server.js'
+import constants from '../../../utils/constants.js'
+const url = constants.routes.TEST_DEVELOPER_SEED_DATA
+
+describe(url, () => {
+  describe('GET', () => {
+    it(`should render the ${url.substring(1)} view`, async () => {
+      const response = await submitGetRequest({ url })
+      expect(response.request.yar.get(constants.redisKeys.FULL_NAME)).toEqual('Test Name')
```
[the remark this answers is not in the document]
Comment (sfitz42) [reply packages/webapp/src/routes/__tests__/test/seed-developer-data.spec.js]: Updated keys to developer journey

File: packages/webapp/src/__mock-data__/test-developer-application.js [hunk 2]
```diff
@@ -0,0 +1,641 @@
+const data = `{
+  "registrationTaskDetails": {
+    "taskList": [
+      {
+        "taskTitle": "Your details",
+        "tasks": [
+          {
+            "title": "Add your details",
+            "status": "COMPLETED",
+            "completedTaskUrl": "/land/check-your-details",
```
[the remark this answers is not in the document]
Comment (sfitz42) [reply packages/webapp/src/__mock-data__/test-developer-application.js]: Removed land owner task list from developer seed data

File: packages/webapp/src/__mock-data__/test-developer-application.js [hunk 3]
```diff
@@ -0,0 +1,641 @@
...
+        ]
+      }
+    ]
+  },
+  "fullname": "Test Name",
+  "role": "Ecologist",
+  "role-other": "",
+  "email-value": "test@test.com",
+  "yes": "yes",
+  "land-boundary": "documentOrImage",
```
[the remark this answers is not in the document]
Comment (sfitz42) [reply packages/webapp/src/__mock-data__/test-developer-application.js]: Removed land owner keys from developer seed data

**Review by usman-j-shaikh — APPROVED** [review]
Seems fine to me.

**Review by usman-j-shaikh — APPROVED** [review]

**Comment by shaundefra** [comment]
tested by leena

**Pull request merged**